### PR TITLE
Added the port for node exporter.

### PR DESCRIPTION
**Pull request by rkruze**

### CODE REVIEW

File: aws/cluster.tf [hunk 1]
```diff
@@ -106,6 +106,14 @@ resource "aws_security_group" "node_sec_group" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+  # node exporter
+  ingress {
+    from_port   = 9100
```
Comment (0xdiba) [inline aws/cluster.tf:111]: Are we sure we want to expose node exporter to the world? the info can probably be scraped by the prometheus instance.

Not that it makes any difference security wise ( the cluster is unprotected anyway ), just curious on the usecase.

Comment (rkruze) [reply aws/cluster.tf:111]: Fair point, it doesn't work without this rule, but perhaps we should update all things that need it to use the security group name vs 0.0.0.0/0.

Comment (rkruze) [reply aws/cluster.tf:111]: Ok, changed it, we now open these ports to the security group and not to the full internet.

**Review by 0xdiba — APPROVED** [review]
Looks good! :rocket:

**Pull request merged**